### DISCUSSION TITLE
How to remove the Application Menu empty panel?

### DISCUSSION THREAD

**Discussion by rulai-yinqiang** [harborsiem/WindowsRibbon#24] [Q&A]
Hello Harborsiem,

Thanks for your promt replies about my questions. I'm relly impressed. I have another quesion:
 
I don't need Recent Items. But there's a Empty Panel. How to I remove it? Or how can I put some big buttons on it?

![image](https://github.com/harborsiem/WindowsRibbon/assets/33405365/adf1a589-9e95-485e-8862-67eb8e232cd8)

**Comment by harborsiem** [comment]
That's by Microsoft design. You can't change it. See also Microsoft docs about Ribbon Framework.
